modules/swagger-codegen/src/main/resources/aspnetcore/Solution.mustache

```diff
@@ -2,7 +2,7 @@
 # Visual Studio 15
 VisualStudioVersion = 15.0.26114.2
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "{{packageName}}", "src\{{packageName}}\{{packageName}}.csproj", "{{packageGuid}}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "{{packageName}}", "src\{{packageName}}\{{packageName}}.csproj", "{{packageGuid}}"
```

Comment:
I think this will have to be conditional based on some flag. We support Xamarin Studio (Visual Studio on Mac, now) as well as Rider.

Reply:
Makes sense, most frustrating part is VS just fails with no real indication of why it couldn't load the project. Maybe the documentation can be updated to let others know about this issue?

thanks for looking into it.

Reply:
What about generating 2 solutions files for Xamarin and VS Studio respectively? e.g.
```
IO.Swagger.xamarin.sln
IO.Swagger.vs.sln
```
Developers can open the one based on their preference.

Reply:
I don't know if that's a good approach as it could be very confusing which to open in other editors like VS Code or Rider.

Because this is dependent on VS 2017, I think it would make sense to switch on that, as there's been discussion about other VS 2017 issues. Or, if there's a project version in the new sln format, that would be ideal, because that format would likely be used for multiple future versions of VS. That is, passing `vs=2017` then opening in VS 2019 could be confusing.

Reply:
What about making the project GUID itself a option to be passed in?